Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealitySpeechCommandsProfileInspector.cs

```diff
@@ -86,7 +87,13 @@ public override void OnInspectorGUI()
             EditorGUILayout.PropertyField(recognizerStartBehaviour);
             EditorGUILayout.PropertyField(recognitionConfidenceLevel);
 
-            RenderList(speechCommands);
+            EditorGUILayout.Space();
+            showSpeechCommands = EditorGUILayout.Foldout(showSpeechCommands, "Speech Commands");
```

Comment:
Why do we need a foldout for this?

It's literally the only thing in this profile. I think it is a bit overboard.

Reply:
It's not about the amount of content but the prevention of context switching.

Reply:
> None of the other input profiles with a similar list was updated.

The speech profile was the first that got changed. The others are in the backlog.

The reason for enabling folding this list is to allow customers to simplify their view when not adding new commands.